crates/runtime/src/trampolines/aarch64.rs

```diff
@@ -0,0 +1,119 @@
+use wasmtime_asm_macros::asm_func;
+
+#[rustfmt::skip]
+asm_func!(
+    "host_to_wasm_trampoline",
+    r#"
+        .cfi_startproc
+        hint #34 // bti c
```

Comment:
I think this may actually be able to be `bti c` instead of `hint #34` since I presume LLVM's internal assembler is a bit more modern than random ancient binutils assemblers.